torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -95,17 +95,20 @@ class TORCH_API TensorExprKernel {
       const std::string& kernel_func_name,
       std::unordered_map<c10::Symbol, NNCLoweringFunction> custom_lowerings =
           {},
+      std::vector<int64_t> symbolic_shape_inputs = {},
       bool pre_alloc = false);
 
   explicit TensorExprKernel(
       const std::shared_ptr<Graph>& subgraph,
       std::unordered_map<c10::Symbol, NNCLoweringFunction> custom_lowerings =
           {},
+      std::vector<int64_t> symbolic_shape_inputs = {},
```

Comment:
I think we need to start adding documentation about all the arguments. We keep piling them on (not your fault :) ), and most of them are used for completely different things - it might be difficult to grasp this code for new people.

Reply:
I added a brief description of the arguments above. We can update it with more details as needed.